packages/ariakit/src/combobox/__examples__/combobox-inline-autocomplete/test.tsx

```diff
@@ -0,0 +1,37 @@
+import { getByRole, press, render, type } from "ariakit-test";
+import { axe } from "jest-axe";
+import Example from ".";
+
+const getCombobox = () => getByRole("combobox");
+const getPopover = () => getByRole("listbox", { hidden: true });
+
+test("a11y", async () => {
+  const { container } = render(<Example />);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+test("autocomplete on arrow down key", async () => {
+  render(<Example />);
+  const combobox = getCombobox();
+  const popover = getPopover();
+  await press.Tab();
+  expect(popover).not.toBeVisible();
+  expect(combobox).toHaveFocus();
+  await type("a");
+  expect(popover).toBeVisible();
+  await press.ArrowDown();
+  expect(combobox).toHaveValue("Apple");
```

Comment:
Some examples may re-mount their elements in future changes (for example, if we start showing the popover conditionally, the element references will be different each time the popover opens), so it's better to query the elements again on each assertion (an exception would be an extensive test suite where multiple queries could impact performance significantly and slow down the test, which is often not the case).

```suggestion
  await press.Tab();
  expect(getPopover()).not.toBeVisible();
  expect(getCombobox()).toHaveFocus();
  await type("a");
  expect(getPopover()).toBeVisible();
  await press.ArrowDown();
  expect(getCombobox()).toHaveValue("Apple");
```

Reply:
done